modules/solid_mechanics/src/materials/CombinedPlasticityStressUpdate.C

```diff
@@ -0,0 +1,258 @@
+  params.addParam<Real>("Q", 0.0, "Saturation value for isotropic hardening (Q in Voce model)");
+  params.addParam<Real>("b", 0.0, "Rate constant for isotropic hardening (b in Voce model)");
+  params.addParam<FunctionName>("yield_stress_function",
+                                "Yield stress as a function of temperature");
+  params.addParam<Real>("yield_stress", "The point at which plastic strain begins accumulating");
+  params.addParam<FunctionName>("isotropic_hardening_function",
+                                "True stress as a function of plastic strain");
+  params.addParam<Real>("isotropic_hardening_constant", "Isotropic hardening slope");
+  params.addCoupledVar("temperature", 0.0, "Coupled Temperature");
+  params.addDeprecatedParam<std::string>(
+      "plastic_prepend",
+      "",
+      "String that is prepended to the plastic_strain Material Property",
+      "This has been replaced by the 'base_name' parameter");
+  params.set<std::string>("effective_inelastic_strain_name") = "effective_plastic_strain";
+  params.addParam<Real>("kinematic_hardening_modulus", 0.0, "Kinematic hardening modulus");
+  params.addParam<Real>("material_constant_gamma",
+                        0.0,
+                        "The nonlinear hardening parameter (gamma) for back stress evolution");
+
+  return params;
+}
+
+template <bool is_ad>
+CombinedPlasticityStressUpdateTempl<is_ad>::CombinedPlasticityStressUpdateTempl(
+    const InputParameters & parameters)
+  : BackstressRadialReturnStressUpdateTempl<is_ad>(parameters),
+    _plastic_prepend(this->template getParam<std::string>("plastic_prepend")),
+    _yield_stress_function(this->isParamValid("yield_stress_function")
+                               ? &this->getFunction("yield_stress_function")
+                               : nullptr),
+    _yield_stress(this->isParamValid("yield_stress") ? this->template getParam<Real>("yield_stress")
+                                                     : 0),
+    _isotropic_hardening_constant(
+        this->isParamValid("isotropic_hardening_constant")
+            ? this->template getParam<Real>("isotropic_hardening_constant")
+            : 0),
+    _isotropic_hardening_function(this->isParamValid("isotropic_hardening_function")
+                                      ? &this->getFunction("isotropic_hardening_function")
+                                      : nullptr),
+    _yield_condition(-1.0), // set to a non-physical value to catch uninitalized yield condition
+    _isotropic_hardening_slope(0.0),
+    _plastic_strain(this->template declareGenericProperty<RankTwoTensor, is_ad>(
+        _base_name + _plastic_prepend + "plastic_strain")),
+    _plastic_strain_old(this->template getMaterialPropertyOld<RankTwoTensor>(
+        _base_name + _plastic_prepend + "plastic_strain")),
+    _backstress(this->template declareGenericProperty<RankTwoTensor, is_ad>("backstress")),
```

Comment:
I was wrong, I _backstress and _backstress_old should be moved to the BackstressRadialReturnClass.  Then in this class you use them as needed and they do not need to be defined in this class, they will be inherited from the parent class.  This will then be like _creep_strain in RadialReturnCreepStressUpdateBase.  PowerLawCreepStressUpdate updates _creep_strain.  You'll just move these constructors to BackstressRadialReturnClass along with the material definitions and make them protected.